youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -274,7 +273,8 @@ def run(self, information):
             raise PostProcessingError('WARNING: unable to obtain file audio codec with ffprobe')
 
         more_opts = []
-        if self._preferredcodec == 'best' or self._preferredcodec == filecodec or (self._preferredcodec == 'm4a' and filecodec == 'aac'):
+        if self._preferredcodec == 'best' or self._preferredcodec == filecodec or (
```

Comment:
Best not to make unrelated changes in a PR. Same issue below. Also these changes often don't obey the linter